### PR TITLE
issue 1247

### PR DESCRIPTION
**Pull request by sapayth**

### CODE REVIEW

File: templates/orders/date-export.php [hunk 1]
```diff
@@ -41,17 +41,19 @@
...
-            <?php
-                wp_nonce_field( 'dokan_vendor_order_export_action', 'dokan_vendor_order_export_nonce' );
-            ?>
-            <input type="submit" name="dokan_order_export_all"  class="dokan-btn dokan-btn-sm dokan-btn-danger dokan-btn-theme" value="<?php esc_attr_e( 'Export All', 'dokan-lite' ); ?>">
-            <input type="submit" name="dokan_order_export_filtered"  class="dokan-btn dokan-btn-sm dokan-btn-danger dokan-btn-theme" value="<?php esc_attr_e( 'Export Filtered', 'dokan-lite' ); ?>">
-            <input type="hidden" name="order_date" value="<?php echo esc_attr( $filter_date ); ?>">
-            <input type="hidden" name="order_status" value="<?php echo esc_attr( $order_status ); ?>">
-        </div>
-    </form>
+    <?php if ( current_user_can( 'administrator' ) || current_user_can( 'seller' ) || current_user_can( 'dokan_export_order' ) ) : ?>
```
Comment (nurul-umbhiya) [inline templates/orders/date-export.php:44]: checking for manage_woocommerce and dokan_export_order is enough

File: includes/Dashboard/Templates/Orders.php [hunk 2]
```diff
@@ -95,7 +95,7 @@ function handle_order_export() {
 
         $post_data = wp_unslash( $_POST );
 
-        if ( ! isset( $post_data['dokan_vendor_order_export_nonce'] ) || ! wp_verify_nonce( sanitize_key( $post_data['dokan_vendor_order_export_nonce'] ), 'dokan_vendor_order_export_action' ) ) {
+        if ( ! isset( $post_data['dokan_vendor_order_export_nonce'] ) || ! wp_verify_nonce( sanitize_key( $post_data['dokan_vendor_order_export_nonce'] ), 'dokan_vendor_order_export_action' ) || ! current_user_can( 'dokan_export_order' ) || ! current_user_can( 'administrator' ) || ! current_user_can( 'seller' ) ) {
```
Comment (nurul-umbhiya) [inline includes/Dashboard/Templates/Orders.php:98]: revert the changes as it was

Comment (sapayth) [reply includes/Dashboard/Templates/Orders.php:98]: Here we are checking if the specific vendor staff has permission for exporting the order list as per our discussion yesterday.
Should we remove the check? Or maybe we could modify it with:
` if ( ! isset( $post_data['dokan_vendor_order_export_nonce'] ) || ! wp_verify_nonce( sanitize_key( $post_data['dokan_vendor_order_export_nonce'] ), 'dokan_vendor_order_export_action' ) || ! current_user_can( 'dokan_export_order' ) || ! current_user_can( 'manage_woocommerce' ) )`

**Comment by shohag121** [comment]
@sapayth Bhai, Kindly add a title and description on this PR.
